### PR TITLE
initial project

### PR DESCRIPTION
**Pull request by JordiGN**
# Description
What did you implemented/Why did you implemented this:
 - Initial setup.
- Create a React-Native application. 
- Correct execution
- This commit is related to issue #2 
 

 # Testing
Download code, enter to project folder run _npm install_ on the console, run _npx react-native start_ and in another console run _npx react-native run-android_

for iOS run _npx pod-install_ and then _npx react-native run-ios_

### CODE REVIEW

File: FlightReservation/App.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ * @flow strict-local
+ */
+
+import React from 'react';
```
Comment (OwenRamire) [inline FlightReservation/App.js:9]: Delete all the code lines or you could leave something like this:
```javascript
import React from 'react';
import {SafeAreaView, Text, StyleSheet} from 'react-native';

const App = () => {
   return (
   <SafeAreaView style={{flex: 1, justifyContent: 'center', alignItems: 'center'}}>
      <Text>Hello world</Text>
   </SafeAreaView>
   );
}

export default App;
```

**Review by OwenRamire — CHANGES_REQUESTED** [review]
When you create your setup, just delete all the things that you think you won't use or just delete the code lines on `App.js` and do:
- `git add .`
- `git commit -m "React Native init"` or `Initial commit` 
on your master branch.

**Review by OwenRamire — APPROVED** [review]

**Review by 1ng-HeberH — APPROVED** [review]

**Review by alcamk — APPROVED** [review]
It's ok for me

**Review by alcamk — APPROVED** [review]
It's ok

**Pull request merged**